source/features/select-notifications.tsx

```diff
@@ -14,18 +14,22 @@ import {
 	GitPullRequestDraftIcon,
 	GitPullRequestIcon,
 	IssueOpenedIcon,
+	RocketIcon,
 	XCircleIcon,
+	XIcon,
 } from '@primer/octicons-react';
 
 import features from '.';
 
 const filters = {
 	'Pull requests': ':is(.octicon-git-pull-request, .octicon-git-pull-request-closed, .octicon-git-pull-request-draft, .octicon-git-merge)',
 	Issues: ':is(.octicon-issue-opened, .octicon-issue-closed)',
-	Open: ':is(.octicon-issue-opened, .octicon-git-pull-request)',
+	Deployments: ':is(.octicon-rocket, .octicon-x)',
+	Open: ':is(.octicon-issue-opened, .octicon-git-pull-request, .octicon-rocket)',
 	Closed: ':is(.octicon-issue-closed, .octicon-git-pull-request-closed)',
 	Draft: '.octicon-git-pull-request-draft',
 	Merged: '.octicon-git-merge',
+	'Run failed': '.octicon-x',
```

Comment:
Is "Run failed" the exact wording GitHub uses? If not, we should try to match it.